week04/assignment04/artist.go

```diff
@@ -0,0 +1,32 @@
+package assignment04
+
+import "fmt"
+
+type Artist struct {
+	Artistname string
```

Comment:
In Go, we use PascalCase for naming, `ArtistName`. However, we also try not to be redundant. This field does not need the `Artist` prefix.

```suggestion
	Name string
```

Reply:
I was getting a "Field and method with the same name" error when I use Name as the field name.